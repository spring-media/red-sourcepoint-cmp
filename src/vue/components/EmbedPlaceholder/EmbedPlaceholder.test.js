import { mount } from '@vue/test-utils';
import { EmbedPlaceholder } from './index';

describe('EmbedPlaceholder', () => {
  it('should render without any errors', () => {
    expect(mount(EmbedPlaceholder).element).toMatchSnapshot();
  });
});
