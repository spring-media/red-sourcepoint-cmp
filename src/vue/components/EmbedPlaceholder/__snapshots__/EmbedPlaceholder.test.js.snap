// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmbedPlaceholder should render without any errors 1`] = `
<div
  class="embed-placeholder__container"
>
  <div
    class="embed-placeholder__header"
  >
    <svg
      fill="none"
      height="84"
      width="100%"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        cx="16"
        cy="16"
        fill="#E9ECEF"
        r="16"
      />
       
      <rect
        fill="#E9ECEF"
        height="10"
        width="60"
        x="40"
        y="7"
      />
       
      <rect
        fill="#E9ECEF"
        height="6"
        width="40"
        x="40"
        y="19"
      />
       
      <rect
        fill="#E9ECEF"
        height="8"
        width="100%"
        y="44"
      />
       
      <rect
        fill="#E9ECEF"
        height="8"
        width="100%"
        y="60"
      />
       
      <rect
        fill="#E9ECEF"
        height="8"
        width="75%"
        y="76"
      />
    </svg>
  </div>
   
  <div
    class="embed-placeholder__headline"
  >
    
            An dieser Stelle findest du Inhalte aus Sozialen Netzwerken
        
  </div>
   
  <div
    class="embed-placeholder__description"
  >
    
            Um mit Inhalten aus Sozialen Netzwerken zu interagieren oder diese darzustellen, brauchen wir deine Zustimmung.
        
  </div>
   
  <button
    class="embed-placeholder__button"
  >
    Soziale Netzwerke aktivieren
  </button>
   
  <div
    class="embed-placeholder__footer-text"
  >
    
            Ich bin damit einverstanden, dass mir externe Inhalte aus Sozialen Netzwerken angezeigt werden.
            Damit können personenbezogene Daten an Drittanbieter übermittelt werden.
            Mehr dazu findest du in der 
    <a
      class="embed-placeholder__text-link"
      href="#"
      rel="noopener"
      target="_blank"
    >
      Beschreibung dieses Datenverarbeitungszweck
    </a>
     und in den Datenschutzinformationen dieser 
    <a
      class="embed-placeholder__text-link"
      href="#"
      rel="noopener"
      target="_blank"
    >
      Drittanbieter
    </a>
    .
        
  </div>
</div>
`;
